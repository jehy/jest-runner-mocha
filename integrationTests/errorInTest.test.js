'use strict';

const runJest = require('./runJest');

it('Works when it has an error in the tests', () => {
  return expect(runJest('errorInTest')).resolves.toMatchSnapshot();
});
