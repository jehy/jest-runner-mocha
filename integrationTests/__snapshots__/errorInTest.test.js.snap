// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Works when it has an error in the tests 1`] = `
"FAIL integrationTests/__fixtures__/errorInTest/__mocha__/__tests__/file.test.js ● Test suite failed to run Some error message 2 | const assert = require('assert'); 3 | > 4 | throw new Error('Some error message'); | ^ 5 | 6 | describe('My tests', () => { 7 | it('This is a test', () => { at mocked-stack-trace Test Suites: 1 failed, 1 total Tests: 0 total Snapshots: 0 total Time: Ran all test suites.
"
`;
